Doc/library/os.rst

```diff
@@ -878,6 +901,9 @@ as internal buffering of data.
 
    .. availability:: Unix.
 
+      The function is a limited on Emscripten and WASI, see
```

Comment:
Did you mean:
```suggestion
      The function is limited on Emscripten and WASI, see
```
or is there a missing word after "limited"?

There are several more occurrences of this.

Reply:
Fixed, thanks